installer/bundle/bundle_skel_PPC-RHEL.sh

```diff
@@ -0,0 +1,584 @@
+            shift 1
+            ;;
+
+        --install)
+            verifyNoInstallationOption
+            installMode=I
+            shift 1
+            ;;
+
+        --purge)
+            verifyNoInstallationOption
+            installMode=P
+            shouldexit=true
+            shift 1
+            ;;
+
+        --remove)
+            verifyNoInstallationOption
+            installMode=R
+            shouldexit=true
+            shift 1
+            ;;
+
+        --restart-deps)
+            restartDependencies=--restart-deps
+            shift 1
+            ;;
+
+        --source-references)
+            source_references
+            cleanup_and_exit 0
+            ;;
+
+        --upgrade)
+            verifyNoInstallationOption
+            installMode=U
+            shift 1
+            ;;
+
+        --version)
+            echo "Version: `getVersionNumber $OM_PKG scx-`"
+            exit 0
+            ;;
+
+        --version-check)
+            printf '%-15s%-15s%-15s%-15s\n\n' Package Installed Available Install?
+
+            # omi
+            versionInstalled=`getInstalledVersion omi`
+            versionAvailable=`getVersionNumber $OMI_PKG omi-`
+            if shouldInstall_omi; then shouldInstall="Yes"; else shouldInstall="No"; fi
+            printf '%-15s%-15s%-15s%-15s\n' omi $versionInstalled $versionAvailable $shouldInstall
+
+            # scx
+            versionInstalled=`getInstalledVersion scx`
+            versionAvailable=`getVersionNumber $OM_PKG scx-cimprov-`
+            if shouldInstall_scx; then shouldInstall="Yes"; else shouldInstall="No"; fi
+            printf '%-15s%-15s%-15s%-15s\n' scx $versionInstalled $versionAvailable $shouldInstall
+
+            exit 0
+            ;;
+
+        --debug)
+            echo "Starting shell debug mode." >&2
+            echo "" >&2
+            echo "SCRIPT_INDIRECT: $SCRIPT_INDIRECT" >&2
+            echo "SCRIPT_DIR:      $SCRIPT_DIR" >&2
+            echo "EXTRACT DIR:     $EXTRACT_DIR" >&2
+            echo "SCRIPT:          $SCRIPT" >&2
+            echo >&2
+            set -x
+            shift 1
+            ;;
+
+        -? | --help)
+            usage `basename $0` >&2
+            cleanup_and_exit 0
+            ;;
+
+        *)
+            usage `basename $0` >&2
+            cleanup_and_exit 1
+            ;;
+    esac
+done
+
+if [ -z "${installMode}" ]; then
+    echo "$0: No options specified, specify --help for help" >&2
+    cleanup_and_exit 3
+fi
+
+#
+# Note: From this point, we're in a temporary directory. This aids in cleanup
+# from bundled packages in our package (we just remove the diretory when done).
+#
+
+mkdir -p $EXTRACT_DIR
+cd $EXTRACT_DIR
+
+# Do we need to remove the package?
+if [ "$installMode" = "R" -o "$installMode" = "P" ]
+then
+    if [ -f /opt/microsoft/scx/bin/uninstall ]; then
+        /opt/microsoft/scx/bin/uninstall $installMode
+    else
+        # This is an old kit.  Let's remove each separate provider package
```

Comment:
So PPC is a brand new platform, and it has no notion of an "old kit". As such, you don't need this else case; the uninstall script will ALWAYS exit.  I think you can delete 436, 438-451.

We had this because, in the olden days, there was not an "uninstall" script. We added one due to product requirements (PM wanted a simple uninstall script), and then it turned out to be the right thing to just call it from here (avoid code duplication).

BUT: If you were upgrading from an old version without an uninstall script, we needed a fallback.

In this case, there is no "old" PPC kit for AIX. Thus, you don't need this at all.